src/main/java/com/github/seratch/jslack/api/model/block/composition/MarkdownTextObject.java

```diff
@@ -13,5 +13,5 @@
 public class MarkdownTextObject extends TextObject {
     private final String type = "mrkdwn";
     private String text;
-    private Boolean verbatim;
+    private boolean verbatim;
```

Comment:
The documentation of the Slack API states that the `verbatim` field is optional (similarly the `emoji` field from `PlainTextObject` is also optional). The API examples always render the `emoji` field (as `true`, but that is its default value)--so that is not helpful. I picked the `Boolean` because basically you have 3 possible states:
- true
- false
- not present (and therefore not rendered in the resulting JSON sent to the Slack API)

Reply:
Right, fair enough. I will revert this change.

Reply:
Will update the code with a short comment